### PR TITLE
2022 is the year of multi-modality

### PR DESCRIPTION
**Pull request by LysandreJik**
Long overdue update of the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,14 +48,16 @@ limitations under the License.
 </h4>
 
 <h3 align="center">
-    <p>State-of-the-art Natural Language Processing for Jax, PyTorch and TensorFlow</p>
+    <p>State-of-the-art Machine Learning for Jax, PyTorch and TensorFlow</p>
```
Comment (patil-suraj) [inline README.md:51]:
```suggestion
    <p>State-of-the-art Machine Learning for JAX, PyTorch and TensorFlow</p>
```

File: README.md [hunk 2]
```diff
@@ -48,14 +48,16 @@ limitations under the License.
...
 </h3>
 
 <h3 align="center">
     <a href="https://hf.co/course"><img src="https://raw.githubusercontent.com/huggingface/transformers/master/docs/source/imgs/course_banner.png"></a>
 </h3>
 
-🤗 Transformers provides thousands of pretrained models to perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. Its aim is to make cutting-edge NLP easier to use for everyone.
+🤗 Transformers provides thousands of pretrained models to perform tasks on different modalities such as Natural Language Processing, Computer Vision, and Audio.
```
Comment (patil-suraj) [inline README.md:58]: (nit) 
```suggestion
🤗 Transformers provides thousands of pretrained models to perform tasks on different modalities such as Natural Language, Vision, and Audio. 
```

File: README.md [hunk 3]
```diff
@@ -65,7 +67,7 @@ limitations under the License.
 
 You can test most of our models directly on their pages from the [model hub](https://huggingface.co/models). We also offer [private model hosting, versioning, & an inference API](https://huggingface.co/pricing) for public and private models.
 
-Here are a few examples:
+Here are a few examples, in NLP:
```
Comment (patil-suraj) [inline README.md:70]:
```suggestion
Here are a few examples:

 in NLP:
```

File: README.md [hunk 4]
```diff
@@ -142,19 +153,19 @@ The model itself is a regular [Pytorch `nn.Module`](https://pytorch.org/docs/sta
...
 1. Lower compute costs, smaller carbon footprint:
     - Researchers can share trained models instead of always retraining.
     - Practitioners can reduce compute time and production costs.
-    - Dozens of architectures with over 2,000 pretrained models, some in more than 100 languages.
+    - Dozens of architectures with over 20,000 pretrained models, some in more than 100 languages.
 
 1. Choose the right framework for every part of a model's lifetime:
     - Train state-of-the-art models in 3 lines of code.
-    - Move a single model between TF2.0/PyTorch frameworks at will.
+    - Move a single model between TF2.0/PyTorch/Jax frameworks at will.
```
Comment (patil-suraj) [inline README.md:168]:
```suggestion
    - Move a single model between TF2.0/PyTorch/JAX frameworks at will.
```

File: README.md [hunk 5]
```diff
@@ -48,14 +48,16 @@ limitations under the License.
...
 </h3>
 
 <h3 align="center">
     <a href="https://hf.co/course"><img src="https://raw.githubusercontent.com/huggingface/transformers/master/docs/source/imgs/course_banner.png"></a>
 </h3>
 
-🤗 Transformers provides thousands of pretrained models to perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. Its aim is to make cutting-edge NLP easier to use for everyone.
+🤗 Transformers provides thousands of pretrained models to perform tasks on different modalities such as Natural Language Processing, Computer Vision, and Audio. 
+
+These models can perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. They can perform tasks on images such as object detection, image segmentation, optical character recognition. They can perform tasks on audio such as speech recognition, audio classification and speech to text.
```
Comment (patil-suraj) [inline README.md:60]: speech recognition and speech to text are the same tasks, no ?

**Review by patil-suraj — APPROVED** [review]
Thanks a lot for updating this!

File: README.md (hunk 2 above)
Comment (NielsRogge) [inline README.md:58]: NLP is not really a modality (more like a domain in AI), I would either say "on different modalities such as language, vision and audio", or say "perform tasks on various domains of AI such as Natural Language Processing, Computer Vision, Audio.

And maybe also mention that models might involve several modalities at the same time.

File: README.md (hunk 5 above)
Comment (NielsRogge) [inline README.md:60]:
```suggestion
These models can perform tasks on texts such as text classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. They can perform tasks on images such as image classification, object detection, segmentation. They can perform tasks on audio such as speech recognition, audio classification and speech-to-text. They can perform tasks on several modalities combined, such as optical character recognition, information extraction from scanned documents, auto-encoding videos, visual question answering, ...
```
Some of these are soon to arrive :D

File: README.md [hunk 6]
```diff
@@ -74,6 +76,15 @@ Here are a few examples:
...
 - [Translation with T5](https://huggingface.co/t5-base?text=My+name+is+Wolfgang+and+I+live+in+Berlin)
 
+In Computer Vision:
+- [Image classification with ViT](https://huggingface.co/google/vit-base-patch16-224)
+- [Object Detection with DETR](https://huggingface.co/facebook/detr-resnet-50)
+- [Image Segmentation with DETR](https://huggingface.co/facebook/detr-resnet-50-panoptic)
+
+In Audio:
+- [Automatic Speech Recognition with Wav2Vec2](https://huggingface.co/facebook/wav2vec2-base-960h)
+- [Keyword Spotting with Wav2Vec2](https://huggingface.co/superb/wav2vec2-base-superb-sid)
```
Comment (anton-l) [inline README.md:86]:
```suggestion
- [Keyword Spotting with Wav2Vec2](https://huggingface.co/superb/wav2vec2-base-superb-ks)
```

File: README.md (hunk 5 above)
Comment (anton-l) [reply README.md:60]:
```suggestion
These models can perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. They can perform tasks on images such as object detection, image segmentation, optical character recognition. They can perform tasks on audio such as speech recognition, audio classification and text-to-speech.
```
Soon to be true too ;)

**Review by anton-l — APPROVED** [review]
🔥2️⃣0️⃣2️⃣2️⃣🔥

File: README.md [hunk 7]
```diff
@@ -48,14 +48,16 @@ limitations under the License.
...
 
 <h3 align="center">
     <a href="https://hf.co/course"><img src="https://raw.githubusercontent.com/huggingface/transformers/master/docs/source/imgs/course_banner.png"></a>
 </h3>
 
-🤗 Transformers provides thousands of pretrained models to perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. Its aim is to make cutting-edge NLP easier to use for everyone.
+🤗 Transformers provides thousands of pretrained models to perform tasks on different modalities such as language, vision, and audio. 
+
+These models can perform tasks on texts such as text classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. They can perform tasks on images such as image classification, object detection, segmentation. They can perform tasks on audio such as speech recognition and audio classification. They can perform tasks on several modalities combined, such as table question answering, optical character recognition, information extraction from scanned documents, video classification, visual question answering.
```
Comment (lewtun) [inline README.md:60]: This paragraph has a lot of repeated "They can perform tasks". What do you think about rephrasing it a bit like the following? (Just an idea)

```suggestion
These models can applied on:

* 📝 Text, for text classification, information extraction, question answering, summarization, translation, text generation, and more in over 100 languages. 
* 🖼️ Images, for image classification, object detection, and segmentation. 
* 🗣️ Audio, for speech recognition and audio classification. 

Transformer models can also perform tasks on **several modalities combined**, such as table question answering, optical character recognition, information extraction from scanned documents, video classification, and visual question answering.
```

**Review by lewtun — COMMENTED** [review]
Super exciting revamp of the README ❤️ !

I added a few suggestions to tighten the text a bit (feel free to ignore if you disagree)

File: README.md (hunk 7 above)
Comment (LysandreJik) [reply README.md:60]: I love it! The only issue I see is that with the original phrasing, it's open-ended: "tasks like ..." which also mean that tasks not included here are also handled (which is the case!). In the proposed wording, it seems like it's limited to the task mentioned. If you can rephrase your proposal to take this into account, I'm happy to merge it!

File: docs/source/index.mdx [hunk 8]
```diff
@@ -31,29 +34,26 @@ other libraries developed by Hugging Face and the Hub.
...
-- 8 architectures with over 30 pretrained models, some in more than 100 languages
-
-Choose the right framework for every part of a model's lifetime:
-
-- Train state-of-the-art models in 3 lines of code
-- Deep interoperability between Jax, Pytorch and TensorFlow models
-- Move a single model between Jax/PyTorch/TensorFlow frameworks at will
-- Seamlessly pick the right framework for training, evaluation, production
-
-The support for Jax is still experimental (with a few models right now), expect to see it grow in the coming months!
```
Comment (LysandreJik) [inline docs/source/index.mdx:56]: I removed this @patil-suraj, @patrickvonplaten, I think JAX support is way more stable now than it was a few months ago, right?

File: README.md (hunk 7 above)
Comment (lewtun) [reply README.md:60]: I tweaked the text to include "for tasks such as" in each bullet point. It's a bit repetitive, but maybe the use of a list and emojis is the main improvement here :)

**Review by sgugger — APPROVED** [review]
Thanks for the update!

**Review by julien-c — APPROVED** [review]

**Comment by LysandreJik** [comment]
Thank you all for your contributions to the README!

**Pull request merged**